lib/gracenote/search.rb

```diff
@@ -1,9 +1,15 @@
 module Gracenote
   module Search
     # @params mode [String] 'SINGLE_BEST' or 'SINGLE_BEST_COVER'
-    def search(artist: nil, album: nil, track: nil, mode: nil)
+    def search(artist: nil, album: nil, track: nil, mode: nil, range: nil)
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for search is too high. [18.03/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Cyclomatic complexity for search is too high. [7/6]
* Method has too many lines. [12/10] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)